eng/pipelines/vmr-build-full.yml

```diff
@@ -0,0 +1,50 @@
+# This yml is used by these pipelines and triggers:
+#
+# - installer-unified-build-full (public)
+#   - PR: full
+
+trigger: none
+pr:
+  branches:
+    include:
+    - main
+    - release/*
+    exclude:
+    - release/*.0.2xx
+    - release/*.0.3xx
+    - release/*.0.4xx
+
+parameters:
+- name: vmrBranch
+  displayName: dotnet/dotnet branch to push to
+  type: string
+  default: ' '
+
+variables:
+- ${{ if ne(parameters.vmrBranch, ' ') }}:
+  - name: VmrBranch
+    value: ${{ replace(parameters.vmrBranch, ' ', '') }}
+- ${{ else }}:
+  - name: VmrBranch
+    value: ${{ replace(replace(variables['System.PullRequest.TargetBranch'], 'refs/heads/', ''), 'refs/pull/', '') }}
+
+# enable source-only build for pipelines with the -source-build suffix
+- name: isSourceOnlyBuild
+  value: ${{ contains(variables['Build.DefinitionName'], '-source-build') }}
```

Comment:
When @akoeplinger merged the different YMLs he added this pattern so that multiple pipelines can use the same YML. Any objection on doing this here as well?

1. Remove this YML again
2. Update the vmr-build.yml and add this condition: `contains(variables['Build.DefintionName'], '-full')`

This would allow to add two new comment trigger only pipelines:
- installer-source-build-full
- installer-unified-build-full

Reply:
Ahh yeah, that's not a bad way of doing it. Will do.